splinepy/microstructure/tiles/double_lattice.py

```diff
@@ -75,30 +96,33 @@ def create_tile(
             # Constant auxiliary values
             if i_derivative == 0:
                 cl = contact_length
-                pp = parameters[0, 0]  # parameters.shape == [1]
+                ppv = parameters[0, 0]  # parameters.shape == [1]
+                ppd = parameters[0, index_second_value]
                 v_one_half = 0.5
                 v_one = 1.0
                 v_zero = 0.0
             else:
                 cl = 0.0
-                pp = parameter_sensitivities[0, 0, i_derivative - 1]
+                ppv = parameter_sensitivities[0, 0, i_derivative - 1]
+                ppd = parameter_sensitivities[
+                    0, index_second_value, i_derivative - 1
+                ]
```

Comment:
Please document better. a simple name change can do wonders `thickness_center`?

Reply:
what about thick_vert_hor and thick_diagonal?